### DISCUSSION TITLE
BEST for talking with PDF documents

### DISCUSSION THREAD

**Discussion by kalle07** [janhq/jan#2388] [General]
i tryed a lot, at least 6 open source projects ...
Jan has made it !!!
very fast and uses my Nvidia RTX !!!
THY !!!

**Comment by davidjimenez75** [comment]
> i tryed a lot, at least 6 open source projects ...
> Jan has made it !!!
> very fast and uses my Nvidia RTX !!!
> THY !!!

What RTX are you using? I'm still trying to make it work with my 3060 12GB 😓

**Reply by kalle07** [reply]
should be working, 
why you think it doesnt work?

**Reply by davidjimenez75** [reply]
The issue was that my XEON E5-2670 v1 lacked the Intel® AVX2 extensions.
